doc/getting_started/index.rst

```diff
@@ -128,77 +95,36 @@ Next, install additional Python packages required by Zephyr in a shell or
    # macOS and Windows
    pip3 install -r zephyr/scripts/requirements.txt
 
-Some notes on pip's ``--user`` option:
-
-- Installing with ``--user`` is the default behavior on Debian-based
-  distributions and is generally recommended on Linux to avoid conflicts with
-  Python packages installed using the system package manager.
-
-- On Linux, verify the Python user install directory ``~/.local/bin`` is at the front of
-  your PATH environment variable, otherwise installed packages won't be
-  found.
-
-- On macOS, Homebrew disables the ``--user`` flag\ [#homebrew_user]_.
-
-- On Windows using ``cmd.exe``, although it's possible to use the ``--user``
-  flag, it makes it harder for the command prompt to find executables installed
-  by pip.
-
-
 Set Up a Toolchain
 ******************
 
-.. note::
+Zephyr binaries are compiled and linked by a *toolchain*\
+[#tools_native_posix]_. You now need to *install* and *configure* a toolchain.
```

Comment:
From the perspective of "Getting Started as fast as possible", the native posix and host toolchains options are awesome because they _skip_ completely a big and time-consuming step: no need to install any SDK. That's why these options were highlighted this early. I could be wrong but I'm afraid you're making these options much easier to miss.

Reply:
The problem is that those only apply to Linux users, and also many Zephyr users will want to get up and running on real hardware as soon as possible, this being an embedded RTOS. They probably want to build, flash and run a simple app on a board they have.

Reply:
> That's why these options were highlighted this early.

As Carles says, those options only apply to Linux users. The Linux reader who is going through this document in linear order has already installed the Zephyr SDK, since that's in installation_linux.rst, or chosen to go against its recommendations. So: this section is skippable by those who follow recommendations, and those who don't have already decided they know better than us and can be trusted to read the details.

> I could be wrong but I'm afraid you're making these options much easier to miss.

I'm not so sure.

I've watched over the shoulder as experienced Linux users followed this guide; several of them skipped reading any and all text and just started blindly copy/pasting the first terminal commands they could find. So less may be more here.

Further, I think that having a choice which clearly says "Host Toolchains" is a pretty clear signal to this sort of user that they have choices for skipping that option.

Reply:
> The Linux reader who is going through this document in linear order has already installed the Zephyr SDK, since that's in installation_linux.rst, or chosen to go against its recommendations

You're right, I missed that. Much easier to spot now that I know about https://builds.zephyrproject.org/zephyrproject-rtos/zephyr/16064/docs/getting_started/index.html !